.build-test-rules.yml

```diff
@@ -3,7 +3,7 @@ examples:
   disable:
     - if: CONFIG_NAME in ["esp-box", "esp-box-lite"]
       reason: Do not build examples for deprecated BSPs
-    - if: (IDF_VERSION_MAJOR == 5 and IDF_VERSION_MINOR < 3) and CONFIG_NAME in ["esp32_p4_function_ev_board", "esp-box-3"]
+    - if: (IDF_VERSION_MAJOR == 5 and IDF_VERSION_MINOR < 3) and CONFIG_NAME in ["esp32_p4_function_ev_board", "esp-box-3", "esp_bsp_generic", "esp_bsp_devkit"]
       reason: Example depends on BSP, which is supported only for IDF >= 5.3
```

Comment:
esp_bsp_devkit's dependency is `  idf: ">=5.2"`, seems there is some inconsistency

Reply:
Fixed, it should be supported from IDF 5.2